src/components/PeoplePage.tsx

```diff
@@ -1,8 +1,105 @@
+/* eslint-disable max-len */
```

Comment:
Do you need this eslint? Try to remove it and fix a problem without it. It will be a good practice